app/services/station_service.rb

```diff
@@ -0,0 +1,8 @@
+class StationService
+
+  def self.get_nearest_station(location)
+    conn = Faraday.new(url: "https://developer.nrel.gov/api/alt-fuel-stations/v1/nearest.json")
+    response = conn.get("?limit=1&location=#{location}&api_key=#{ENV['nrel_api_key']}")
```

Comment:
Nice work taking advantage of query parameters so that you don't need to retrieve more than 1 station!

In general, it's often more readable to put these query parameters into a block and use `faraday.params` to add the data. However, during a timed assessment this is a totally valid solution.

Reply:
Check out the Faraday docs to see how they add query parameters in a block to learn more about this syntax